requirements.txt

```diff
@@ -16,6 +16,8 @@ seaborn>=0.9.0
 colorcet>=2.0.1
 tqdm
 python-igraph>=0.7.1
+python-louvain>=0.16
+cdlib>=0.2.6
```

Comment:
make sure this specification is right

Reply:
check this issue. https://github.com/aristoteleo/dynamo-release/issues/431

Please also fix all issues from that report